Portal/src/Datahub.Core/Services/TranslationService.cs

```diff
@@ -11,7 +11,7 @@ public TranslationService(IConfiguration configuration)
     {
         var useFreeApi = configuration.GetSection("DeepL").GetValue("UseFreeApi", true);
         var authKey = configuration.GetSection("DeepL").GetValue<string>("AuthKey");
-        _client = new Translator(authKey, new TranslatorOptions() { ServerUrl = useFreeApi ? "https://api-free.deepl.com" : null });
+        _client = new Translator(authKey!, new TranslatorOptions() { ServerUrl = useFreeApi ? "https://api-free.deepl.com" : null });
```

Comment:
We may not need to specify the server url at all; according to the Deepl-dotnet documentation it will auto select the correct api url based on the user's account. It's not critical but we can add a TODO to revisit it later